Cargo.toml

```
@@ -1,38 +1,24 @@
 [package]
 name = "libp2p-community-tor"
-version = "0.3.0-alpha"
+version = "0.2.0"
```

```suggestion
version = "0.4.0"
```
I think we should go forwards in the version.